### ISSUE TITLE
Box widget collapses but reopens straight away

### ISSUE THREAD

**Issue by Xoduz** [Insolita/yii2-adminlte-widgets#4]
When I use the collapse controls on a Box widget in yii2-adminlte-widgets v1.1.1, the box collapses - then reopens again immediately. If (without any further clicking on collapse controls) I now navigate away to a different page and back to the same page, the Box widget will now be in the collapsed state. If I expand it, it will open and then collapse again. Navigate away to a different page and back, and it will now be in the uncollapsed state.

Basically it seems to save the collapse state, but only displays the new state properly after refreshing the page.

**Comment by Insolita** [comment]
ok, unfortunately i can check it next week only
